vnext/Microsoft.ReactNative/Views/TouchEventHandler.cpp

```diff
@@ -432,27 +458,37 @@ void TouchEventHandler::DispatchTouchEvent(TouchEventType eventType, size_t poin
   } else
 #endif // USE_FABRIC
   {
-    auto touches = folly::dynamic::array();
+    winrt::Microsoft::ReactNative::JSValueArray touches;
     for (const auto &pointer : m_pointers) {
-      folly::dynamic touch = GetPointerJson(pointer, pointer.target);
-      touches.push_back(touch);
+      touches.push_back(GetPointerJson(pointer, pointer.target));
     }
 
     // Package up parameters and invoke the JS event emitter
-    const char *eventName = GetTouchEventTypeName(eventType);
+    const wchar_t *eventName = GetTouchEventTypeName(eventType);
     if (eventName == nullptr)
       return;
-    folly::dynamic params = folly::dynamic::array(eventName, std::move(touches), std::move(changedIndices));
 
-    m_context->CallJSFunction("RCTEventEmitter", "receiveTouches", std::move(params));
+    const auto paramsWriter = MakeJSValueArgWriter(eventName, std::move(touches), std::move(changedIndices));
```

Comment:
It maybe missing a namespace - in case if it compiles, then we may drop the long namespace in other cases too. It is probably worth to add `using namespace winrt::Microsoft::ReactNative` in the top of the file.